src/main/java/com/example/bookstore/dto/request/BookRequestDto.java

```diff
@@ -1,13 +1,21 @@
 package com.example.bookstore.dto.request;
 
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 public class BookRequestDto {
+    @NotBlank
     private String title;
+    @NotBlank
     private String author;
+    @NotBlank
     private String isbn;
```

Comment:
ISBN has other business rules too, so please validate it. You can you built-in annotation for this or create your custom.